packages/slate-react/src/components/editable.tsx

```diff
@@ -733,14 +733,25 @@ export const Editable = (props: EditableProps) => {
           onClick={useCallback(
             (event: React.MouseEvent<HTMLDivElement>) => {
               if (
-                !readOnly &&
                 hasTarget(editor, event.target) &&
                 !isEventHandled(event, attributes.onClick) &&
                 isDOMNode(event.target)
               ) {
                 const node = ReactEditor.toSlateNode(editor, event.target)
                 const path = ReactEditor.findPath(editor, node)
 
+                if (event.detail === 3 && readOnly) {
```

Comment:
In general, I prefer an enum so I don't have to remember what 3 is... even if it's just the line before, e.g. `const TRIPLE_CLICK = 3` and then `if (event.detail === TRIPLE_CLICK ...`

Reply:
ok